upstream.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = SDL2_net-2.0.1
+
+source_url = https://www.libsdl.org/projects/SDL_net/release/SDL2_net-2.0.1
```

Comment:
I tried downloading a file from this URL and...

```
$ unzip SDL2_net-2.0.1.zip 
Archive:  SDL2_net-2.0.1.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of SDL2_net-2.0.1.zip or
        SDL2_net-2.0.1.zip.zip, and cannot find SDL2_net-2.0.1.zip.ZIP, period.
```

Could you try https://www.libsdl.org/projects/SDL_net/release/SDL2_net-2.0.1.zip instead?